tests/api/test_dtls.c

```diff
@@ -1474,6 +1474,86 @@ int test_dtls_drop_client_ack(void)
     return EXPECT_RESULT();
 }
 
+int test_dtls_bogus_finished_epoch_zero(void)
+{
+    EXPECT_DECLS;
+#if defined(HAVE_MANUAL_MEMIO_TESTS_DEPENDENCIES) && defined(WOLFSSL_DTLS) && \
+    !defined(WOLFSSL_NO_TLS12) && !defined(NO_PSK) && defined(HAVE_AES_CBC) && \
+    defined(WOLFSSL_AES_128) && !defined(NO_SHA256) && defined(WOLFSSL_STATIC_PSK)
```

Comment:
Is the PSK ciphersuite really necessary for this test? This severely limits the scope of configurations that this will be tested in.